src/Taonomics.md

```diff
@@ -6,54 +6,35 @@ Tao is the token for Bittensor and gatekeeps access, representing bandwidth to t
 
 
 
-# Emission Split
+**Emission Split**: Each subnetwork \\(h\\) receives a set percentage \\(E_h\\) of total emission \\(E=\sum_h E_h\\).
+Servers and Validators evenly split emissions per block, with the most emissions issued to the best-performing Servers and largest Validators.
 
-Servers and Validators evenly split emissions per block, with the most emissions issued to the best-performing Servers.
+**Weights**: Each Validator \\(i\\) evaluates the relative utility of each Server \\(j\\), and sets weights \\(\sum_jW_{ij}=1\\) on-chain, signed with Validator hotkey. A relatively high weight indicates relatively high Server utility.
 
+**Prerank**: Server rank \\(R_j\\) is the sum of weighted stake set on \\(j\\), before consensus. A high rank indicates that majority stake agrees on relatively high utility of a Server.
+\\[P_j = \sum_i S_i \cdot W_{ij}\\]
 
-# Validation and B (Bonds)
+**Consensus**: On-chain consensus then calculates stake-based median weight \\(C_j\\) for each Server \\(j\\), which is the maximum weight supported by at least \\(\kappa\\)-majority stake, so that stake sum (as ratio of total active stake) of active Validators setting at least consensus weight is \\(\kappa\\) (typically \\(\kappa=0.5\\)).
```

Comment:
stake sum of active Validators setting at least consensus weight
-> stake sum of active Validators setting **at most** consensus weight